core/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -498,9 +498,45 @@ trait VacuumCommandImpl extends DeltaCommand {
       basePath: Path,
       relativizeIgnoreError: Boolean,
       isBloomFiltered: Boolean): Seq[String] = {
-    getActionRelativePath(action, fs, basePath, relativizeIgnoreError).map { relativePath =>
+    val paths = getActionRelativePath(action, fs, basePath, relativizeIgnoreError).map {
+      relativePath =>
       Seq(relativePath) ++ getAllSubdirs("/", relativePath, fs)
     }.getOrElse(Seq.empty)
+
+    val deletionVectorPath =
+      getDeletionVectorRelativePath(action, fs, basePath, relativizeIgnoreError).map(pathToString)
+
+    paths ++ deletionVectorPath.toSeq
+  }
+
+    /** Returns the relative path of the on-disk deletion vector. */
```

Comment:
Indent. Also:
```suggestion
    /** Returns the path of the on-disk deletion vector if it is stored relative to the `basePath` otherwise `None`. */
```